mail-filter/bmf/files/bmf-0.9.4_parameter-not-declared-fix.patch

```diff
@@ -0,0 +1,24 @@
+Bug: https://bugs.gentoo.org/885043
```

Comment:
please submit it upstream & link here

Reply:
No. I need to figure out how to send patch through sourceforge.net

Reply:
Also, Which slot of sys-libs/db should be used?